workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -194,73 +127,28 @@ internal static IEnumerable<BuildTarget> GetUnityBuildTargets(SpatialBuildPlatfo
                 var result = BuildPipeline.BuildPlayer(buildPlayerOptions);
                 if (result.summary.result != BuildResult.Succeeded)
                 {
-                    throw new BuildFailedException($"Build failed for {workerPlatform}");
+                    throw new BuildFailedException($"Build failed for {workerType}");
                 }
 
-                var zipPath = Path.GetFullPath(Path.Combine(PlayerBuildDirectory, workerBuildData.PackageName));
-
-                var basePath = PathUtil.Combine(BuildPaths.BuildScratchDirectory, workerBuildData.PackageName);
-
-                Zip(zipPath, basePath,
-                    targetEnvironment == BuildEnvironment.Local
-                        ? PlayerCompression.Disabled
-                        : PlayerCompression.Enabled);
+                var zipPath = Path.Combine(PlayerBuildDirectory, workerBuildData.PackageName);
+                var basePath = PathUtils.Combine(PathUtils.BuildScratchDirectory, workerBuildData.PackageName);
+                Zip(zipPath, basePath, targetEnvironment == BuildEnvironment.Local);
```

Comment:
This now compresses if the build env is local instead of when its cloud. Was that intended?

Reply:
+1 looks like a bug

Reply:
yep, that is a bug